charts/openstack-cloud-controller-manager/values.yaml

```diff
@@ -46,6 +46,8 @@ livenessProbe: {}
 # Set readinessProbe in the same way like livenessProbe
 readinessProbe: {}
 
+# set dnsPolicy to 'ClusterFirstWithHostNet' if the cluster is deployed behind a corporate reverse proxy
+dnsPolicy: default
```

Comment:
I think I'd default to `ClusterFirstWithHostNet` as it makes more sense. This will require adding a release note to the PR description, that the default behavior is changed.

Reply:
i would leave the default behavior and offer the option to be modified ONLY if necessary

Reply:
But `dnsPolicy` should be set to `ClusterFirstWithHostNet` by default if `hostNetwork` is set to `true`.